### PR TITLE
Add expiry date edge case tests

### PR DESCRIPTION
**Pull request by okhan-okbay-cko**
Covering expiry date edge cases.

### CODE REVIEW

File: iOS Example Frame SPM/iOS Example Frame Regression Tests/Helpers/StaticTexts.swift [hunk 1]
```diff
@@ -10,5 +10,7 @@ import Foundation
 
 enum StaticTexts {
     static let cardNumberNotValidError = "Please enter a valid card number"
+    static let expiryDateIsInThePastError = "Expiry date is in the past"
```
Comment (deepesh-vasthimal-cko) [inline iOS Example Frame SPM/iOS Example Frame Regression Tests/Helpers/StaticTexts.swift:13]: Are these static text not part of the SDK itself?

**Review by deepesh-vasthimal-cko — APPROVED** [review]
